cosmwasm/README.md

```diff
@@ -1,45 +1,92 @@
-# Deploy
+# Intro
 
-First build the contracts
+This folder contains pyth contract on CosmWasm and utilities to deploy it in CosmWasm chains.
 
+# Deployment
```

Comment:
use `##` for sections of the doc (like this). Only use `#` for the title of the readme.

You probably have to indent the headings below one additional level as well.

Reply:
I see. thanks!